.github/workflows/instance.yml

```
@@ -0,0 +1,23 @@
+name: instance
```

💅 Suggest a name that will be nice to see in the GH UI:

```suggestion
name: Create tflocal instance
```